lib/mas/tasks/firms.rake

```diff
@@ -0,0 +1,14 @@
+namespace :firms do
+  desc 'Index all existing firms'
+  task :index => :environment do
+    puts 'Do you want to index all existing firms? [type `yes` to confirm]'
+    confirmation = STDIN.gets.chomp
+    if confirmation.downcase == 'yes'
+      puts 'Building firms index...'
+      Firm.all.each {|f| IndexFirmJob.perform_later(f) }
```

Comment:
totally agree, but I've already merged it after seeing a dog, ship, bike and helmet

will come back to it if we encounter any problems :)